jenkins_pipelines/environments/manager-TEST-reportdb-hub-tests

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([cron('H 19 * * *')]),
```

Comment:
If I may suggest, I think that for developing and debugging issues it might be better to just don't trigger new runs automatically.

Reply:
@mcalmer what do you think?

Reply:
We do this always for our test environments. Setup in the night  and at the next morning your have a fresh setup with the latest code from yesterday evening. But maybe a little bit later. The time is also UTC. 
7pm is 6pm and in some weeks 5pm. That would trigger the resetup still during the typically presence time of us:-)